tools/tpm2_dump_capability.c

```diff
@@ -600,7 +600,7 @@ dump_tpm_capability (TPMU_CAPABILITIES    *capabilities,
     case TPM_CAP_COMMANDS:
         dump_command_attr_array (capabilities->command.commandAttributes,
                                  capabilities->command.count);
-        /* fallthrough */
+        break;
```

Comment:
@webmeister nailed it: the bug never manifest due to the caller ignoring the return result. This means the errant behavior was never visible to users invoking this tool. I figured I'd save the git history an extra commit and just lump these two changes together since I found them both at the same time and they're fixing a common warning.

I'll defer to the maintainer / reviewer on this one.